### DISCUSSION TITLE
Embeddings selection in Styles

### DISCUSSION THREAD

**Discussion by Rellik-cloud** [Acly/krita-ai-diffusion#1121] [Ideas]
So far everything works perfectly but I'm missing my embeddings to work specifically negative embeddings personally, I would like to be able to select negative or positive embeddings under the "style prompt" section like we are able to select the loras in the "LoRA" section

**Comment by Acly** [comment]
You can use embeddings by putting them in the prompt. For example `embedding:EasyNegative` to use EasyNegative.safetensors 

Unlike LoRA it matters where they are inserted into the text, so it's better to do it there directly.

**Reply by Hamsterun** [reply]
And embeddings should be in the folder ComfyUI > models > embeddings, like normal?

**Reply by Dead5nake** [reply]
I hope you figured this out in January lol. I tried the files in both these locations, the embeddings worked in:

\AppData\Roaming\krita\ai_diffusion\server\ComfyUI\models\embeddings
or
\AppData\Roaming\krita\ai_diffusion\server\models\embeddings
or both.
